utils/hooks/useKeyboardListeners.ts

```diff
@@ -0,0 +1,27 @@
+import {useEffect, useState} from 'react';
+import {Keyboard, Platform} from 'react-native';
+
+const useKeyboardListeners: () => boolean = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    const keyboardShowListener = Keyboard.addListener(
+      Platform.OS === 'ios' ? 'keyboardWillShow' : 'keyboardDidShow',
+      () => setIsVisible(true),
+    );
+
+    const keyboardHideListener = Keyboard.addListener(
+      Platform.OS === 'ios' ? 'keyboardWillHide' : 'keyboardDidHide',
```

Comment:
This check means that all platforms except `ios` will use `keyboardDidHide` hence Android and Windows Phones